teuthology/config.py

```diff
@@ -37,7 +37,7 @@ def load(self, conf=None):
                 self._conf = yaml.safe_load(conf)
             return
         if os.path.exists(self.yaml_path):
-            self._conf = yaml.safe_load(file(self.yaml_path))
+            self._conf = yaml.safe_load(open(self.yaml_path))
```

Comment:
probably it's not in the scope of this change, could you add a commit to
```py
with open(self.yaml_path) as f:
  self._conf = yaml.safe_load(f)
```
?

Reply:
done